Sources/OpenTracing/Span.swift

```diff
@@ -1,23 +1,57 @@
 import Foundation
 
-/// - TODO: headerdoc
+/// Represents information related to an event with a timespan
 public protocol Span {
+
+    /// The span context that refers to this span
```

Comment:
Unrelated to docs, but have we considered making this a computed property instead?

Reply:
we can do that